generated/nidigital/nidigital/enums.py

```diff
@@ -63,6 +63,32 @@ class PinState(Enum):
     NOT_A_PIN_STATE = 254
     PIN_STATE_NOT_ACQUIRED = 255
 
+    def __str__(self):
+        if self.name == 'ZERO':
+            return str('0')
+        elif self.name == 'ONE':
+            return str('1')
+        elif self.name == 'L':
+            return str('L')
+        elif self.name == 'H':
+            return str('H')
+        elif self.name == 'X':
+            return str('X')
+        elif self.name == 'M':
+            return str('M')
+        elif self.name == 'V':
+            return str('V')
+        elif self.name == 'D':
+            return str('D')
+        elif self.name == 'E':
+            return str('E')
```

Comment:
I think you can adjust the code generator so that these cases are covered under:
```
        else:
            return str(self.name)
```